### PR TITLE
[MM-43920] - Set Workspace Inactivity Email to only fire once and change subject

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
This PR makes changes such that we only send the inactivity email once.
Screenshot 2022-05-04 at 11 37 39

#### Ticket Link
[MM-43920](https://mattermost.atlassian.net/browse/MM-43920)

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: app/server_inactivity.go [hunk 1]
```diff
@@ -27,58 +32,22 @@ func (s *Server) doInactivityCheck() {
 		return
 	}
 
+	_, sysValErr := s.Store.System().GetByName(INACTIVITY_EMAIL_SENT)
```
Comment (neallred) [inline app/server_inactivity.go:35]: Do we need to keep checking the `INACTIVITY` entry to not resend to installations who have already seen the email once?

**Review by neallred — APPROVED** [review]
LGTM, thanks @AGMETEOR

**Review by marianunez — APPROVED** [review]
Thanks @AGMETEOR

**Comment by marianunez** [comment]
Adding @dontoisme to review the footer and if we should include the instructions to disable in the systems settings part:

```
To disable future inactivity emails, set [EmailSettings.EnableInactivityEmail to off in your config.json file]([https://docs.mattermost.com/configure/configuration-settings.html#disable-inactive-server-email-notification](https://docs.mattermost.com/configure/configuration-settings.html#enable-insecure-outgoing-connections)s).”
```

See https://mattermost.atlassian.net/browse/MM-43920?focusedCommentId=126886

**Comment by dontoisme** [comment]
I'm good keeping the footer this way.

**Review by dontoisme — APPROVED** [review]

**Review by stevemudie — APPROVED** [review]
Thanks to @AGMETEOR's help I received the one-time email and it was the new format and looking good.

No further emails showed up as expected.

**Pull request merged**